solutions/src/test/java/ru/sberbank/school/task10/TestFixedThreadPool.java

```diff
@@ -0,0 +1,71 @@
+package ru.sberbank.school.task10;
+
+import org.junit.After;
+import org.junit.jupiter.api.Assertions;
+import org.junit.Before;
+import org.junit.Test;
+
+
+public class TestFixedThreadPool {
+
+ //   private FixedThreadPool pool;
+    private ScalableThreadPool pool;
+    private final Object lock = new Object();
+    private int count;
+    private int thread_1;
+    private int thread_2;
+    private int thread_3;
+
+    @Before
+    public void initialize() {
+       pool = new ScalableThreadPool(3, 5);
```

Comment:
&#39;method def&#39; child has incorrect indentation level 7, expected level should be 8.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a907806540>*